app/views/api/_observation.xml.builder

```diff
@@ -18,7 +18,14 @@ xml.tag!(tag,
   xml_detailed_object(xml, :consensus_name, object.name)
   xml_confidence_level(xml, :confidence, object.vote_cache)
   if detail
-    xml_html_string(xml, :notes, object.notes.to_s.tpl_nodiv)
+    xml.notes do
```

Comment:
Re CodeClimate issues.  Will deal with those tomorrow.
Re promotion.  You're right. I had this my notes, but don't know what I was thinking.